spec/services/atmosphere/cloud/vmt_updater_spec.rb

```diff
@@ -75,13 +75,13 @@
   end
 
   it 'does not update VMT--AT relation when VMT is old' do
-    at, source_vmt, source_cs = at_with_vmt_on_cs
-    target_cs = create(:compute_site)
+    at, source_vmt, source_t = at_with_vmt_on_tenant
```

Comment:
Useless assignment to variable - `at`. Use `_` or `_at` as a variable name to indicate that it won't be used.